### DISCUSSION TITLE
Why are catalogued object images showing up in the figure viewer?

### DISCUSSION THREAD

**Discussion by zsofiaj** [thegetty/quire#926] [Get Help]
In our Italian Paintings catalogue, on each entry page, when I click on a figure in the entry, the lightbox view of the figure images also includes an image of the catalogued object as the last figure. For example, in this entry, which has 4 figures, if you click on fig. 1, the lightbox viewer shows 5 images: the 4 figures, plus the catalogued object:

https://italian-paintings-live.netlify.app/catalogue/volume-1/the-florentine-school/14/#fig-14-1]

This doesn't seem right. @Erin-Cecele I feel like I may have asked you about this before, but I cant' find the relevant question or answer anywhere, so maybe I just meant to ask you! I wondered if this is just how Quire was built to work, but I checked in a few of the Getty catalogues, and those do not include the catalogued object image in the lightbox view. What could be causing this?

And I'm sorry I've posted so many times here over the last couple of weeks! I hope it is not too much. Thank you!

**Comment by Erin-Cecele** [comment]
Apologies for the delay @zsofiaj - looking into this now and will get back to you shortly.

**Comment by Erin-Cecele** [comment]
@zsofiaj—I checked with the team. This is not technically a bug. While there was never an explicit decision about whether lightboxes should omit or include catalogued objects on `entry` pages, the current logic of the lightbox view including ALL images on the page matches the handling of `essay` pages.

Is this handling creating specific issues for you that might make us rethink whether catalogued object images should be included in the lightbox?

**Comment by zsofiaj** [comment]
It just felt odd to encounter the catalogued image there, at the end of the lighbox view, after seeing all of the comparative figures. We thought it seemed like a mistake. We've chosen not to include a caption over our catalogued object images, so it could be confusing to our readers when it shows up there at the end, with no caption. We went back and forth on whether to include a caption or not, but our curator/author wanted to display the main image without the overlaid caption. And at the top of the entry readers have the full tombstone information, so they know what they are looking at if they click on the image from there. But when they encounter the image at the end of the lighbox view, they might be a little lost. 

I had also looked at some older Getty Quire books, and I found at least a couple that don't include the catalogued object image at the end of the lighbox view. Here is one:
https://www.getty.edu/publications/rococo/catalogue/2/

But I do see that in a newer similar catalogue the catalogued object image does appear at the end of the lightbox view:
https://www.getty.edu/publications/french-silver/catalogue/1/#fig-1-16

**Comment by Erin-Cecele** [comment]
This is good feedback; thank you, @zsofiaj. This change was made with the transition to v1, which is why you see a difference between _Rococo_ and _French Silver_. Unfortunately, we don't have the bandwidth to make immediate changes, but this is a good thing for us to keep in mind for the future. 

@geealbers, going out on a limb, but is there a way to suppress an image appearing in a lightbox with custom CSS?

**Reply by geealbers** [reply]
I did a little experimenting and while I was able to suppress the image itself with css, it just left a blank spot in the carousel. They only fix for that would be some javascript work.
